### PR TITLE
Fixes #32398 - add hash-router for tabs

### PR DESCRIPTION
**Pull request by amirfefer**
This makes the new host details tabs routable

### CODE REVIEW

**Comment by xprazak2** [comment]
Works for me. :+1:

**Review by xprazak2 — DISMISSED** [review]

**Comment by MariaAga** [comment]
@amirfefer This is only for the first level of tabs right? not the nested ones

File: webpack/assets/javascripts/react_app/components/HostDetails/consts.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const DEFAULT_TAB = 'Overview';
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/HostDetails/consts.js:1]: Can we use this const here as well?
https://github.com/theforeman/foreman/blob/32492ce2d2742cd5ed495798fcd3d02a6d5de92c/webpack/assets/javascripts/react_app/components/HostDetails/Tabs/index.js#L8

**Comment by amirfefer** [comment]
@MariaAga secondary tabs are handled in plugins:
https://github.com/Katello/katello/pull/9579
https://github.com/theforeman/foreman_ansible/pull/454

**Review by MariaAga — APPROVED** [review]
LGTM

**Comment by MariaAga** [comment]
Thanks @xprazak2 @amirfefer !

**Pull request merged**